src/main/java/org/folio/circulation/domain/representations/logs/CirculationCheckInCheckOutLogEventMapper.java

```diff
@@ -0,0 +1,153 @@
+package org.folio.circulation.domain.representations.logs;
+
+import static java.util.Optional.ofNullable;
+import static org.folio.circulation.support.json.JsonPropertyWriter.write;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.folio.circulation.domain.CheckInContext;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.domain.LoanAndRelatedRecords;
+import org.folio.circulation.domain.UpdatedRequestPair;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CirculationCheckInCheckOutLogEventMapper {
+
+  public static final String CHECK_IN = "CHECK_IN_EVENT";
+  public static final String CHECK_OUT = "CHECK_OUT_EVENT";
```

Comment:
I think events describe things that have already happened and that is better described by using the past tense e.g. `item checked out` or `item checked in`. I think in that style of naming, `event` is redundant.

I believe there are already messages that are published on pub-sub that represent those events. Please make sure that the circulation log specific messages have different names (maybe by including the logging intent in the name).

(This assumes that the existing events do not satisfy the needs of the log)

Reply:
It's possible I misunderstood these constants, and they aren't pub-sub message types, rather a sub-type used in the payload of log messages. Is that the case?

Reply:
Yes, Marc, correct. These are sub-types. General event type - LOG_RECORD_EVENT. But I'll fix it name according to your remark about _EVENT.